<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(
        x1:0, y1:0,
        x2:0, y2:1,
        stop:0 #a8d9d0,
        stop:0.5 #d3e9d4,
        stop:1 #f2e1c9
    );
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 italic 20pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Gestion des ordonnance</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>511</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <column>
     <property name="text">
      <string>CIN</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prenom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Medecin</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>110</y>
      <width>361</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 italic 10pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Ajouter une ordonnance</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>190</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>CIN:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>230</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Prenom:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>280</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Nom:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>320</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Medecin:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>360</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cinEdit">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>190</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="prenomEdit">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>230</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nomEdit">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>280</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="medecinEdit">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>320</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(168, 213, 186);
font: 14pt &quot;Segoe UI&quot;;
color: rgb(51, 51, 51);</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(168, 213, 186);
font: 14pt &quot;Segoe UI&quot;;
color: rgb(51, 51, 51);</string>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>360</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="menu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>211</width>
      <height>731</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(168, 213, 186);</string>
    </property>
    <property name="title">
     <string>menu</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(168, 213, 186);
border-color: rgb(0, 0, 0);
color: rgb(51, 51, 51);
font: 18pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>employe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>290</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(168, 213, 186);
color: rgb(51, 51, 51);
font: 18pt &quot;Segoe UI&quot;;
</string>
     </property>
     <property name="text">
      <string>produit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>400</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(168, 213, 186);
color: rgb(51, 51, 51);
font: 18pt &quot;Segoe UI&quot;;
</string>
     </property>
     <property name="text">
      <string>client</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>510</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(168, 213, 186);
color: rgb(51, 51, 51);
font: 18pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>ordonnance</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>620</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(168, 213, 186);
color: rgb(51, 51, 51);
font: 18pt &quot;Segoe UI&quot;;
</string>
     </property>
     <property name="text">
      <string>fournisseur</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>151</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/552966492_691544266651586_7706130658263486080_n.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
